Control/move_to_pose/move_to_pose.py

```diff
@@ -19,6 +19,10 @@
 Kp_beta = -3
 dt = 0.01
 
+# Robot specifications
+_MAX_LINEAR_SPEED = 15
+_MAX_ANGULAR_SPEED = 7
```

Comment:
Could you please remove the first _ for the two global configurations?

Reply:
OK. it is done.